website/pages/en/help.js

```diff
@@ -47,19 +47,19 @@ class Help extends React.Component {
               <a href="https://github.com/react-navigation/react-navigation/issues">
                 <translate>post an issue</translate>
               </a>{' '}
-              <translate>and be sure to fill out the issue template. If you believe there is a feature missing, please</translate>
+              <translate>and be sure to fill out the issue template. If you believe there is a feature missing, please</translate>{' '}
               <a href="https://react-navigation.canny.io/feature-requests">
                 <translate> create a feature request on Canny</translate>
               </a>
-              <translate>, or if you're feeling up for the task of proposing an API for the feature, </translate>
+              <translate>, or if you're feeling up for the task of proposing an API for the feature, </translate>{' '}
               <a href="https://github.com/react-navigation/rfcs">
                 <translate>submit a RFC!</translate>
-              </a>
+              </a>{' '}
               <translate> If you just need some help, try joining us in the</translate>
-              <code> react-navigation </code>
+              <code>react-navigation</code>
```

Comment:
Thanks for removing the spaces. But now we need to add them to the text around "react-navigation" because now there are no spaces around the word at all.